system-test/storage.ts

```diff
@@ -209,17 +209,12 @@ describe('storage', () => {
         bucket = storageWithoutAuth.bucket('gcp-public-data-landsat');
       });
 
-      it('should list and download a file', done => {
-        bucket.getFiles(
-          {
-            autoPaginate: false,
-          },
-          (err, files) => {
-            assert.ifError(err);
-            const file = files![0];
-            file.download(done);
-          }
-        );
+      it('should list and download a file', async () => {
+        const [files] = await bucket.getFiles({autoPaginate: false});
+        const file = files[0];
+        const [isPblc] = await file.isPublic();
```

Comment:
Would it be okay to use the full `isPublic` word? I could see other maintainers and contributors getting into a typo battle with this var name.

Reply:
Sure, done